KOGITO-2973: Set default container images in TestResources

Sgitario opened the pull request:
JIRA: https://issues.redhat.com/browse/KOGITO-2973
Description: The images for containers are set at pom level. However, when running the tests via IDE, these images are not injected. This task is about to default the images.
When running the tests via maven, it won't make any change. But when running the tests via IDE, it will continue working as expected.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

Sgitario commented:
Depends on https://github.com/kiegroup/kogito-apps/pull/379

cristianonicolai commented:
@Sgitario it seems that we're using the wrong variable setting in the pom.xml. I replaced `<systemProperties>` with `<systemPropertyVariables>` and that made tests work fine on IntelliJ.

See https://youtrack.jetbrains.com/issue/IDEA-101185

Sgitario commented:
> @Sgitario it seems that we're using the wrong variable setting in the pom.xml. I replaced `<systemProperties>` with `<systemPropertyVariables>` and that made tests work fine on IntelliJ.
> 
> See https://youtrack.jetbrains.com/issue/IDEA-101185

I've already replaced the systemProperties to systemPropertyVariables and seems working fine.
